test/tests.bats

```diff
@@ -1194,16 +1191,16 @@ load 'test_helper/bats-assert/load'
   assert_success
 }
 @test "checking setup.sh: setup.sh debug fail2ban" {
-  
+
   run docker exec mail_fail2ban /bin/sh -c "fail2ban-client set dovecot banip 192.0.66.4"
   run docker exec mail_fail2ban /bin/sh -c "fail2ban-client set dovecot banip 192.0.66.5"
-  sleep 10
+  sleep 2
```

Comment:
Is this change the Problem? Although the error with Travis happens on a very different spot..